test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -370,6 +370,21 @@ def test_successful_capture_with_mdd_fields
     assert_successful_response(capture)
   end
 
+  def test_merchant_description
+    merchant_options = {
+      merchantInformation: {
+        merchantDescriptor: {
+          name: 'Test Name',
+          address1: '123 Main Dr',
+          locality: 'Durham',
+        }
+      }
+    }
+
+    assert response = @gateway.purchase(@amount, @credit_card, @options.merge(merchant_options))
+    assert_successful_response(response)
```

Comment:
The fields added in this PR: `name`, `address1` and `locality` are not returned from CyberSource with a successful transaction from what I read, but please weigh in if there are other cases that should be tested for.

Reply:
It seems pretty common for gateways to not return information about passed fields; I don't think you need to worry about it.